app/controllers/smoke_detectors_controller.rb

```diff
@@ -83,20 +83,6 @@ def token_regen
     redirect_to params[:redirect] || smoke_detector_mine_path
   end
 
-  # Used by Helios to verify new tokens
-  def check_token
-    token = SmokeDetector.where(access_token: params[:token]).first
-    payload = {
-      exists: token.present?,
-      owner_name: token&.user&.username,
-      location: token&.location,
-      created_at: token&.created_at,
-      updated_at: token&.updated_at
-    }
-
-    render json: payload
-  end
```

Comment:
I forget when this was introduced, but I think we will want to keep this unless helios is official dead.